### ISSUE TITLE
Update disabled to true and button doest not work

### ISSUE THREAD

**Issue by luisfuertes** [UdaySravanK/RNSwipeButton#50]
**Describe the bug**
If load screen with button with disabled prop "true" and update to disabled prop to "false", styles updated but button swipe functionality not.

**To Reproduce**
1. Load screen with disabled to "true"
2. On `React.useEffect` or with timeout set disabled to "false" 
3. Styles updated to enabled button but if you try to swipe button, it does not work.

**Versions**
```
    "react": "16.13.1",
    "react-native": "0.63.3",
    "rn-swipe-button": "^1.3.2"
```

**Comment by luisfuertes** [comment]
I try to add `key={`swipe-button-${disabled}`}` to button, but get a warning to update state of unmounted component.

**Comment by luisfuertes** [comment]
My button code:
```
import React from 'react'
import PropTypes from 'prop-types'
import { View, ActivityIndicator, Dimensions } from 'react-native'
import SwipeButton from 'rn-swipe-button'
import styles, { buttonColor, buttonDisabledColor, activityIndicatorColor, buttonDisabledIconColor } from './styles'
import Icon from 'react-native-vector-icons/MaterialIcons'

const SlideButton = ({ onSwipeSuccess, label, width, height, isFetching, disabled }) => {
  return (
    <SwipeButton
      //key={`swipe-button-${disabled}`}
      title={label}
      onSwipeSuccess={onSwipeSuccess}
      width={width}
      height={height}
      containerStyles={styles.container}
      titleStyles={disabled ? { ...styles.title, ...styles.titleDisabled } : styles.title}
      thumbIconBackgroundColor={'transparent'}
      thumbIconBorderColor={'transparent'}
      thumbIconComponent={() => <ThumbIconComponent isFetching={isFetching} disabled={disabled} />}
      railBackgroundColor={buttonColor}
      railBorderColor={disabled ? buttonDisabledColor : buttonColor}
      railFillBackgroundColor={'transparent'}
      railFillBorderColor={'transparent'}
      shouldResetAfterSuccess={true}
      resetAfterSuccessAnimDelay={0}
      disabled={disabled}
      disabledRailBackgroundColor={buttonDisabledColor}
      disabledThumbIconBackgroundColor={'transparent'}
      disabledThumbIconBorderColor={'transparent'}
    />
  )
}

SlideButton.defaultProps = {
  width: Dimensions.get('window').width * 0.7,
  height: 48,
  onSwipeSuccess: () => {},
}

SlideButton.propTypes = {
  width: PropTypes.number,
  height: PropTypes.number,
  label: PropTypes.string,
  onSwipeSuccess: PropTypes.func,
  isFetching: PropTypes.bool,
  disabled: PropTypes.bool,
}

export default SlideButton

const ThumbIconComponent = ({ isFetching, disabled }) => (
  <View>
    {isFetching ? (
      <ActivityIndicator size='small' color={disabled ? buttonDisabledIconColor : activityIndicatorColor} />
    ) : (
      <Icon name='keyboard-arrow-right' color={disabled ? buttonDisabledIconColor : 'white'} size={28} />
    )}
  </View>
)

ThumbIconComponent.propTypes = {
  isFetching: PropTypes.bool,
  disbled: PropTypes.bool,
}

```

**Comment by UdaySravanK** [comment]
Hi @luisfuertes , This issue has been fixed in latest version [v1.3.3](https://github.com/UdaySravanK/RNSwipeButton/releases/tag/v1.3.3)
Please check. Thanks for reporting the issue.
